test/anchor-test.js

```diff
@@ -171,4 +174,70 @@ describe("Anchor", () => {
       expect(a).to.have.property("href", "mailto:padded@expressen.se");
     });
   });
+
+  describe("navigates", () => {
+    let browser, anchors;
+    beforeEach("a DOM with anchors", async () => {
+      browser = await new Browser(app).navigateTo("/link-1");
+
+      anchors = browser.document.getElementsByTagName("a");
+    });
+
+    it("clicking relative link navigates to that url", async () => {
+      browser = await anchors[1].click();
+
+      expect(browser.window.location.pathname).to.equal("/link-2");
+      expect(browser.window.location.search).to.equal("?q=a");
```

Comment:
en assert likt den på rad 202 vore bra för övertydlighet!

Reply:
menar du `href` eller `textContent`? den senare kräver nämligen att jag behöver skriva om genereringen av sidan för att skicka in `req` vilket är lite fult